template/src/pages/buy-me-coffee/ContractDemo.tsx

```diff
@@ -12,13 +12,19 @@ export default function BuyMeCoffeeContractDemo() {
 
   return (
     <div className="mt-10 grid grid-cols-1 items-stretch justify-start md:grid-cols-2CoffeeMd md:gap-9 lg:grid-cols-2CoffeeLg">
-      <div>
-        <div className="mb-6 flex justify-start">
-          <h2 className="mb-1 text-4xl font-bold">Messages</h2>
+      <div className="rounded-[20px] border border-solid border-boat-color-palette-line bg-boat-color-palette-backgroundalternate p-[40px]">
+        <div className="relative inline-flex flex-[0_0_auto] flex-col items-start gap-[16px]">
+          <div className="relative inline-flex flex-[0_0_auto] items-center gap-[16px]">
```

Comment:
Let's try to have good HTML semantic, have too many `div` is a bit disorienting.